src/framework/draw/types/geometry.h

```diff
@@ -49,6 +50,18 @@ inline bool isEqual(T a1, T a2)
     }
 }
 
+// coordinate (x, y)
+template<typename T>
+using coord_t = number_t<T>;
+using coord = coord_t<double>;
+using coordi = coord_t<int>;
+
+// distance (w, h)
+template<typename T>
+using dist_t = number_t<T>;
+using dist = dist_t<double>;
+using disti = dist_t<int>;
```

Comment:
I would like more typed values, because... It’s not clear what’s what.
But I agree, it's difficult to type a value of coordinates and distances.

removed coord_t and dist_t